docs/conf.py

```diff
@@ -9,6 +9,7 @@
 import datetime
 
 sys.path.insert(0, os.path.abspath(".."))
+sys.path.insert(0, os.path.abspath("../adafruit_emc2101"))
```

Comment:
Did you have issues building the docs locally without this?  This shouldn't be needed.